frontend/src/pages/Profile/UserSkills/UserSkills.tsx

```diff
@@ -1,3 +1,4 @@
+/* eslint-disable @typescript-eslint/no-unused-vars */
```

Comment:
Is it necessary to use eslint-disable? Do we have unused vars here?

Reply:
Fixed